Source/d_main.c

```diff
@@ -2230,7 +2243,7 @@ void D_DoomMain(void)
         {
           I_StartTic ();
           D_ProcessEvents ();
-          G_BuildTiccmd (&netcmds[consoleplayer][maketic%BACKUPTICS]);
```

Comment:
What is the `[maketic%BACKUPTICS]` part and why was it dropped?

Reply:
netcmds declaration is different in MBF, I replace it with Choco version.